masterlist.yaml

```
@@ -2832,7 +2832,7 @@ plugins:
     after: [ 'DynDOLOD.esp' ]
     msg: [ *runxLODGen ]
 
-  - name: 'ParallaxGen.esp'
+  - name: 'PG_\d+\.esp'
```

Should keep the old name as well, for anyone still using that file.
```suggestion
  - name: 'P(arallaxGen|G_\d+)\.esp'
```